app/scripts/services/bindingModalUtils.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BindingModalUtils', function() {
```

Comment:
@benjaminapetersen We're planning to switch to the angular-patternfly pf-wizard component, which should handle most of this for us.